posix-configs/SITL/init/lpe/hippocampus

```diff
@@ -13,7 +13,7 @@ param set CAL_MAG0_ID 196616
 simulator start -s
 
 pwm_out_sim mode_pwm
-mixer load /dev/pwm_output0 ROMFS/px4fmu_common/mixers/uuv_quad_x.main.mix
+mixer load /dev/pwm_output0 ROMFS/px4fmu_test/mixers/uuv_quad_x.mix
```

Comment:
Does this really make a difference? Supposedly px4fmu_test mixers are for tests only, but you have a real vehicle mixer though.